src/ontology/mondo-edit.obo

```diff
@@ -548928,9 +548928,10 @@ property_value: IAO:0000233 "https://github.com/monarch-initiative/mondo/issues/
 
 [Term]
 id: MONDO:0700285
-name: DMD-related musculodystrophy
+name: DMD-related muscular dystrophy
 subset: inferred_rare
 subset: rare
+synonym: "'DMD-related musculodystrophy'" EXACT []
```

Comment:
The synonym should not have the apostrophe. 
Instead of 
'DMD-related musculodystrophy'
it should be 
DMD-related musculodystrophy